otx/cli/utils/multi_gpu.py

```diff
@@ -169,6 +169,7 @@ def run_child_process(train_func: Callable, rank: int, gpu_ids: List[int], outpu
             output_path (str): output path where task output are saved.
             multi_gpu_port (str): port for communication between multi GPU processes.
         """
+        mp.set_start_method(None, True)  # set multi porcess method as default
```

Comment:
So, what does this line is doing? None? True?

Reply:
This code set multi process method as default by force.
In the Unix, default multi process method is "fork". So, it's set as "fork".

Reply:
Yeah, but your intention is well hidden.
mp.set_start_method(method="fork", force=True) <- Isn't it better?

Reply:
You're right. I'll update it.